spec/models/assignment_spec.rb

```diff
@@ -2135,7 +2135,8 @@ def grouping_count(groupings)
 
           expect(summary).to_not be_empty
           expect(summary[0]).to_not be_empty
-          expect(summary[0]).to include('User name', 'Group', 'Final grade')
+          expect(summary[0]).to include('Group name', 'User Name', 'Last Name', 'First Name', 'Section', 'ID number',
```

Comment:
Oh I suppose I missed this one, but please use the `human_attribute_name` internalization, like the other places you changed.